### PR TITLE
Add Sensu Go supported platforms

### LINKED ISSUE #594: Define platforms for Sensu Go
Create a new page at /getting-started/platforms with:

- A list of supported platforms for Sensu Go, similar to [this list](https://docs.sensu.io/sensu-core/1.4/faq/#sensu-frequently-asked-questions)

Related:
- https://packagecloud.io/sensu/beta
- https://coreos.com/etcd/docs/latest/op-guide/supported-platform.html
- https://github.com/sensu/sensu-enterprise-go/issues/86

### PR DESCRIPTION
**Pull request by apaskulin**
## Description

Create a page listing supported platforms for Sensu Go

Sources:
- https://wiki.ubuntu.com/Releases
- https://golang.org/doc/install
- https://golang.org/doc/install/source
- https://access.redhat.com/articles/3078
- https://www.debian.org/releases/
- https://wiki.debian.org/LTS/

## Motivation and Context

Closes #594 

To do:

- [x] Change filename to platforms

### CODE REVIEW

File: content/sensu-go/5.0/getting-started/supported-platforms.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+| Platform & Version | 64bit | 32bit | | |
+|--------------------|-------|-------|---|---|
+| CentOS 5           | ✅     |      | | |
+| CentOS 6           | ✅     |      | | |
+| CentOS 7           | ✅     |      | | |
+| Ubuntu 16.04       | ✅     |      | | |
+| Ubuntu 16.10       | ✅     |      | | |
+| Ubuntu 17.04       | ✅     |      | | |
+| Ubuntu 17.10       | ✅     |      | | |
```
Comment (grepory) [inline content/sensu-go/5.0/getting-started/supported-platforms.md]: 16.10 -> 17.10 are EOL'd and unsupported.

File: content/sensu-go/5.0/getting-started/supported-platforms.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+| Platform & Version | 64bit | 32bit | | |
+|--------------------|-------|-------|---|---|
+| CentOS 5           | ✅     | ✅    | | |
+| CentOS 6           | ✅     | ✅    | | |
+| CentOS 7           | ✅     | ✅    | | |
+| Ubuntu 16.04       | ✅     | ✅    | | |
+| Ubuntu 16.10       | ✅     | ✅    | | |
+| Ubuntu 17.04       | ✅     | ✅    | | |
+| Ubuntu 17.10       | ✅     | ✅    | | |
```
Comment (grepory) [inline content/sensu-go/5.0/getting-started/supported-platforms.md]: Same as above

File: content/sensu-go/5.0/getting-started/supported-platforms.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
```
Comment (grepory) [inline content/sensu-go/5.0/getting-started/supported-platforms.md]: Same as above.

**Review by grepory — CHANGES_REQUESTED** [review]

**Comment by grepory** [comment]
We aren't shipping any 32-bit binaries.

**Comment by grepory** [comment]
Windows packages won't be available until some time in the coming week or two.

**Comment by cwjohnston** [comment]
@apaskulin I think what’s here looks good, but RHEL and Debian are conspicuously absent.  

Our classic docs pair up similar Linux distros like RHEL/Centos and Debian/Ubuntu. Are we paring this down to Centos and Ubuntu for a particular reason?

**Review by cwjohnston — APPROVED** [review]
LGTM 👍

File: content/sensu-go/5.0/getting-started/platforms.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+|--------------------|-------|-------|---|---|
+| CentOS 5/RHEL      | ✅     |    | | |
+| CentOS 6/RHEL      | ✅     |     | | |
+| CentOS 7/RHEL      | ✅     |     | | |
+| Ubuntu 16.04       | ✅     |     | | |
+| Ubuntu 18.04       | ✅     |     | | |
+| Ubuntu 18.10       | ✅     |     | | |
+| Debian 8           | ✅     |     | | |
+| Debian 9           | ✅     |     | | |
+| Windows 7 and later| ✅     |     | | |
```
Comment (amdprophet) [inline content/sensu-go/5.0/getting-started/platforms.md]: I think we should leave `Windows 7 and later` out of the supported OS's for the agent. We haven't done much (or any) testing of the agent on Windows 7. I'm also not sure if anyone will want to run the agent on a desktop OS.

It does make sense to include it for the `sensuctl` tool though.

Thoughts @annaplotkin?

Comment (annaplotkin) [reply content/sensu-go/5.0/getting-started/platforms.md]: +1 on sensuctl and leaving it out of agent 👍 @amdprophet

Comment (apaskulin) [reply content/sensu-go/5.0/getting-started/platforms.md]: Updated!

**Pull request merged**